File: extensions/offlineAds/src/main/java/com/mxplay/offlineads/exo/oma/AdsManager.java


package com.mxplay.offlineads.exo.oma;

import java.util.List;

public interface AdsManager extends BaseManager {
    void start();

    List<Float> getAdCuePoints();

    void pause();

    void resume();

    void skip();

    void discardAdBreak();

    /** @deprecated */
    @Deprecated
    void requestNextAdBreak();

}

File: extensions/mxInteractiveMediaAds/src/main/java/com/mxplay/interactivemedia/api/AdsManager.kt
package com.mxplay.interactivemedia.api

interface AdsManager : BaseManager {
    companion object{
        const val PRE_ROLL_POD_INDEX = 0
        const val POST_ROLL_POD_INDEX = -1
    }

    fun start()
    val adCuePoints: List<Float?>?
    fun pause()
    fun resume()
    fun skip()
    fun discardAdBreak()

    @Deprecated("")
    fun requestNextAdBreak()
    fun focus()
}